### ISSUE TITLE
Connection to cantonal services

### ISSUE THREAD

**Issue by dominikponniah** [admin-ch/CovidCertificate-App-Android#165]
In the example of the Canton of Bern, it is possible to download the COVID certificate as a PDF directly via https://be.vacme.ch. This must then be uploaded to the Cert app.

With a connection between vacme and the Cert-App, the individual data could be transferred directly. Thus, an upload of PDF files would no longer be necessary.

Maybe there is already a API which could be used?

**Comment by goebelUB** [comment]
The way VacMe et al get the PDF is by connecting to the BAG/FOPH backend and requesting a new certificate. The backend then returns the PDF.
With the new transfer code feature, the BAG backend no longer returns the PDF to VacMe et al but instead sends the certificate directly to the app that registered that transfer code.

Pros:
- The app does not need to implement many APIs from the many different medical system vendors that are used across the different cantons and different pharmacies, doctors, etc. These implement the BAG API anyway, and simply switch from PDF to transfer code.
- No extra detour via VacMe et al, you get the certificate straight to the app: `Vacme -> BAG -> App` rather than `Vacme -> BAG -> Vacme -> App`.
- Very positive feedback that we got from test centres saying transfer codes sped up things for them :)

Cons:
- Does not work for already issued certs
- User needs to create and submit a transfer code to the vaccination/test centre

TLDR: You raise a good point. I consider this is solved mainly by the transfer codes, but also by improved local PDF import (directly from the file, rather than via the QR code scanner).
